content/en/tracing/profiler/compare_profiles.md

```diff
@@ -0,0 +1,82 @@
+---
+title: Compare Profiles
+kind: documentation
+further_reading:
+    - link: 'tracing/profiler/enabling'
+      tag: 'Documentation'
+      text: 'Enable continuous profiler for your application'
+    - link: 'getting_started/profiler'
+      tag: 'Documentation'
+      text: 'Getting Started with Profiler'
+    - link: 'https://www.datadoghq.com/blog/introducing-datadog-profiling/'
+      tag: 'Blog'
+      text: 'Introducing always-on production profiling in Datadog'
+---
+
+Continuous Profiler can compare two profiles or profile aggregations with each other to help you identify code performance improvements, regressions, and structural changes. You can compare a profile with:
+
+- The same profile at a different time,
```

Comment:
'Same profile at a different time' might be confusing, i'd suggest maybe something like:
- Profiles taken at a different time

Reply:
```suggestion
- Profiles taken at a different time,
```